dashboard/src/pages/EventDetails.vue

```diff
@@ -88,6 +88,7 @@
             class="w-fit"
             icon-right="external-link"
             :link="createAbsoluteUrlFromRoute(event.doc.route)"
+            :disabled="['Draft', 'Cancelled'].includes(event.doc.status)"
```

Comment:
nit: hiding the button with v-if would be better no?

This works too

Reply:
I think disabling it would let users know that the provision is there. If needed can be changed later